### PR TITLE
Strip binaries by default

### PR DESCRIPTION
**Pull request by xhochy**
This removes the need to strip binaries explicitly after compilation.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/activate.bat [hunk 1]
```diff
@@ -1,3 +1,4 @@
+set CARGO_PROFILE_RELEASE_STRIP=symbols
```
[the remark this answers is not in the document]
Comment (xhochy) [reply recipe/activate.bat:1]: This is done directly by `rustc`, see https://doc.rust-lang.org/cargo/reference/profiles.html#strip

[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/activate.bat:1]: In https://github.com/conda-forge/cargo-bundle-licenses-feedstock/pull/19 it was looking for a `strip` executable right?

[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/activate.bat:1]: nvm, it was called explicitly.

**Comment by isuruf** [comment]
While it's good for cli tools, I don't think it's good as a default for libraries.

**Comment by xhochy** [comment]
Then let's keep it in the feedstocks.

**Pull request closed without merging**